clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -1071,6 +1071,16 @@ TEST_F(TokenAnnotatorTest, UnderstandsRequiresClausesAndConcepts) {
                     "concept C = (!Foo<T>) && Bar;");
```

Comment:
So, GitHub won't actually let me leave a comment on the line I want to actually talk about, but I noticed a few lines above this, on line 1066, it uses `EXPECT_EQ` instead of `ASSERT_EQ`, which is my mistake from 4 months ago.
But then I noticed a lot of these mistakes elsewhere in the file.
This is unrelated to this patch here, but do any of you have any opinions on unifying these? As I understand the assert exists to avoid out-of-bound accesses and thus segmentation faults. Is anyone opposed to an NFC commit changing all the EXPECTs into ASSERTs? @llvm/pr-subscribers-clang-format

Reply:
I'm in favor. Just for the correctness.

Reply:
+1. You don't even need to create a pull request for this kind of NFC changes.

Reply:
88d1de5ec64210686d93a90529583505635d257d